<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>itemAvailabilityWorkbench</class>
 <widget class="QWidget" name="itemAvailabilityWorkbench">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item Availability Workbench</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>12</number>
     </property>
     <property name="spacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="ItemCluster" name="_item">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>31</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="_close">
             <property name="text">
              <string>&amp;Close</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>34</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTabWidget" name="_tab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="_availabilityTab">
        <attribute name="title">
         <string>Availability</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QRadioButton" name="_availabilityButton">
             <property name="text">
              <string>Availability</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_runningAvailabilityButton">
             <property name="text">
              <string>Running Availability</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_locationDetailButton">
             <property name="text">
              <string>Location/Lot#/Serial# Detail</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="crmButtonsSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="_availabilityStack">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="_availabilityPage">
            <layout class="QGridLayout" name="gridLayout_20"/>
           </widget>
           <widget class="QWidget" name="_runningAvailabilityPage">
            <layout class="QGridLayout" name="gridLayout_21"/>
           </widget>
           <widget class="QWidget" name="_locationDetailPage">
            <layout class="QGridLayout" name="gridLayout_22"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="_ordersTab">
        <attribute name="title">
         <string>Orders</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="_purchaseOrderItemsButton">
             <property name="text">
              <string>Purchase Orders</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_salesOrderItemsButton">
             <property name="text">
              <string>Sales Orders</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_quoteItemsButton">
             <property name="text">
              <string>Quotes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_customerPricesButton">
             <property name="text">
              <string>Customer Prices</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="_ordersStack">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="_purchaseOrderItemsPage">
            <layout class="QGridLayout" name="gridLayout_28"/>
           </widget>
           <widget class="QWidget" name="_salesOrderItemsPage">
            <layout class="QGridLayout" name="gridLayout_29"/>
           </widget>
           <widget class="QWidget" name="_quoteItemsPage">
            <layout class="QGridLayout" name="gridLayout_30"/>
           </widget>
           <widget class="QWidget" name="_customerPricesPage">
            <layout class="QGridLayout" name="gridLayout_33"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="_historyTab">
        <attribute name="title">
         <string>History</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QRadioButton" name="_inventoryHistoryButton">
             <property name="text">
              <string>Inventory History</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_receivingHistoryButton">
             <property name="text">
              <string>Receiving History</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_salesHistoryButton">
             <property name="text">
              <string>Sales History</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="crmButtonsSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="_historyStack">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="_inventoryHistoryPage">
            <layout class="QGridLayout" name="gridLayout_23"/>
           </widget>
           <widget class="QWidget" name="_receivingHistoryPage">
            <layout class="QGridLayout" name="gridLayout_26"/>
           </widget>
           <widget class="QWidget" name="_salesHistoryPage">
            <layout class="QGridLayout" name="gridLayout_27"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="_itemTab">
        <attribute name="title">
         <string>Item Master</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QStackedWidget" name="_itemStack">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="_itemPage">
            <layout class="QGridLayout" name="gridLayout_31"/>
           </widget>
           <widget class="QWidget" name="_whereUsedPage_2">
            <layout class="QGridLayout" name="gridLayout_32"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="_bomTab">
        <attribute name="title">
         <string>Bill of Materials</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="_costedIndentedBOMButton">
             <property name="text">
              <string>Costed Indented BOM</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_whereUsedButton">
             <property name="text">
              <string>Where Used</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="_bomStack">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="_costedIndentedBOMPage">
            <layout class="QGridLayout" name="gridLayout_24"/>
           </widget>
           <widget class="QWidget" name="_whereUsedPage">
            <layout class="QGridLayout" name="gridLayout_25"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>ItemCluster</class>
   <extends>QWidget</extends>
   <header>itemcluster.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_item</tabstop>
  <tabstop>_tab</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>itemAvailabilityWorkbench</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
